### PR TITLE
EPMDJ-2832: Update service group dashboard

### PR DESCRIPTION
**Pull request by SMPlay**
Issue: EPMDJ-2832

### CODE REVIEW

File: src/pages/service-group-full-page/dashboard/sections/coverage-section/coverage-section.module.scss [hunk 1]
```diff
@@ -1,16 +1,5 @@
...
-    &--type {
-      &--increase {
-        color: map-get($color-palette, green100);
-      }
-      &--decrease {
-        color: map-get($color-palette, red100);
-      }
-    }
-  }
+  display: block;
```
Comment (IliaSolovev) [inline src/pages/service-group-full-page/dashboard/sections/coverage-section/coverage-section.module.scss:4]: is it required?

File: src/pages/service-group-full-page/dashboard/sections/coverage-section/methods-tooltip/methods-tooltip.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import '~common/style-guide/index';
+
+.methods-tooltip {
+  font-family: $semibold-font;
+  font-size: 12px;
+
+  &__covered-methods {
```
Comment (IliaSolovev) [inline src/pages/service-group-full-page/dashboard/sections/coverage-section/methods-tooltip/methods-tooltip.module.scss:7]: it is already div

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/pages/service-group-full-page/dashboard/sections/risks-section.tsx [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+              color={RISKS_TYPES_COLOR.NEW}
+              percent={convertToPercentage(newMethodsCount, total)}
+              icon={<Icons.Add height={16} width={16} />}
+            />
+            <SingleBar
+              width={64}
+              height={128}
+              color={RISKS_TYPES_COLOR.MODIFIED}
+              percent={convertToPercentage(modifiedMethodsCount, total)}
+              icon={<Icons.Edit height={16} width={16} viewBox="0 0 16 15" />}
```
Comment (IliaSolovev) [inline src/pages/service-group-full-page/dashboard/sections/risks-section.tsx:44]: Maybe `viewbox="0 0 16 16"` ? It lok strange

File: src/types/service-group-summary.ts [hunk 4]
```diff
@@ -1,4 +1,7 @@
 import { ArrowType } from 'types/arrow-type';
+import { RisksSummary } from './risks-summary';
```
Comment (IliaSolovev) [inline src/types/service-group-summary.ts:2]: I think that imports should be the same

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

**Pull request merged**